deepchecks/checks/distribution/train_test_label_drift.py

```diff
@@ -0,0 +1,230 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Module contains Train Test Drift check."""
+
+from typing import Tuple, Dict, Callable
+
+import pandas as pd
+
+from deepchecks import Dataset, CheckResult, TrainTestBaseCheck, ConditionResult
+from deepchecks.checks.distribution.plot import drift_score_bar_traces, feature_distribution_traces
+from deepchecks.checks.distribution.dist_utils import preprocess_for_psi, earth_movers_distance, psi
+from deepchecks.utils.typing import Hashable
+import plotly.graph_objects as go
+from plotly.subplots import make_subplots
+
+__all__ = ['TrainTestLabelDrift']
+
+
+class TrainTestLabelDrift(TrainTestBaseCheck):
+    """
+    Calculate label drift between train dataset and test dataset, using statistical measures.
```

Comment:
Somehow this line does not appear in the notebook, maybe rerun and re-commit?

Reply:
Weird, I see it. Rerunning anyway